healthcare/api-client/v1/hl7v2/requirements-test.txt

```diff
@@ -0,0 +1,3 @@
+pytest==5.3.2
+gcp-devrel-py-tools==0.0.16
```

Comment:
Remove these two lines

Reply:
You highlited pytest and gcp-devrel-py-tools. Did you mean to highlight gcp-devrel-py-tools and google-cloud-core? We need pytest don't we?

Reply:
Ah sorry, yes, remove `gcp-devrel-py-tools` and `google-cloud-core`.